docker/docker-entrypoint.sh

```diff
@@ -57,7 +57,7 @@ fi
 if [ -n "$NUXEO_CLID" ]; then
   echo "ENTRYPOINT: Write NUXEO_CLID environment variable to /var/lib/nuxeo/instance.clid"
   # Replace -- by a carriage return
-  NUXEO_CLID="${NUXEO_CLID/--/\\n}"
+  NUXEO_CLID="${NUXEO_CLID//--/\\n}"
```

Comment:
What is this supposed to do? This syntax with `//` is not documented

Reply:
Ok sorry it is documented, I didn't read the bash doc carefully enough.

Reply:
Nor the commit description :smile: https://github.com/nuxeo/nuxeo/pull/4346/commits/e73bf009fe19cc81f9c96c998448c8449e899ceb

Reply:
Hm you mean the extended description hidden behind "..." in the PR pages...